packages/pwa-buildpack/lib/Utilities/addImgOptMiddleware.js

```diff
@@ -11,47 +11,28 @@ try {
     markDepInvalid('apicache', e);
 }
 try {
-    expressSharp = require('@magento/express-sharp');
+    hastily = require('hastily'); // eslint-disable-line
```

Comment:
Doing this in order to throw a custom error if the require fails?

Reply:
Yes!